apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -175,13 +184,16 @@ impl Service<QueryPlannerRequest> for BridgeQueryPlanner {
 #[async_trait]
 impl QueryPlanner for BridgeQueryPlanner {
     async fn get(&self, key: QueryKey) -> Result<QueryPlannerContent, QueryPlannerError> {
-        let selections = self.parse_selections(key.0.clone()).await?;
+        let selections = self
+            .parse_selections(key.0.clone(), self.operation_depth_limit)
```

Comment:
Since this is a `self` function, I don't think we need to pass this parameter. We can just create a local variable (to move) in the `parse_selections` fn.